src/testUtils/TestRig.react.js

```diff
@@ -57,9 +59,14 @@ export default class TestRig {
     var element = this.domNode.find(selector)
     this.fillInElement(element, value)
   }
-  toggleCheckbox(selector) {
-    var element = this.domNode.find(selector) // Selector should be specific to the checkbox
-    $(element).prop('checked', !($(element)[0].checked))
+  toggleElement(selector) {
```

Comment:
so chiming in on the naming. I'd love to embrace how we do clickButton/clickLink. So there are two methods really. 

`toggleElement` should in fact take an element, which is more of a utility method if for whatever we need to get specific. 

Then there is `toggleCheckbox` which probably takes the label text of the checkbox. And this method uses that to figure out the specific element, which it then passes to toggleElement. 

(I'm stealing this whole convention from capybara, one of the nicer things about rails testing).